src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -24,6 +24,7 @@ Author(s):
     X(hstring, WordDelimiters, "wordDelimiters", DEFAULT_WORD_DELIMITERS)                                                                                                                             \
     X(bool, CopyOnSelect, "copyOnSelect", false)                                                                                                                                                      \
     X(bool, FocusFollowMouse, "focusFollowMouse", false)                                                                                                                                              \
+    X(winrt::Microsoft::Terminal::Control::GraphicsAPI, GraphicsAPI, "experimental.rendering.graphicsAPI")                                                                                            \

```

Comment:
we may also remove `experimental.` from these, if you want - if we're gonna keep them for a while...